drivers/platform/maxim/common/maxim_pwm.c

```diff
@@ -0,0 +1,355 @@
+/***************************************************************************//**
+ *   @file   maxim_pwm.c
+ *   @brief  Implementation of MAX32655 PWM driver.
```

Comment:
Since this driver will be common to all supported Maxim MCUs, delete the mention of MAX32655.